docs/src/tty.rst

```diff
@@ -98,3 +115,25 @@ API
     Gets the current Window size. On success it returns 0.
 
 .. seealso:: The :c:type:`uv_stream_t` API functions also apply.
+
+.. c:function:: int uv_tty_set_vterm_state(uv_tty_vtermstate_t state)
+
+    Set whether console virtual terminal sequences are processed by libuv or
+    console.
+    This call is not necessary because new Windows10 consoles are usually
+    detected automatically. But a call is required when using ANSI X3.64 and
+    Xterm 256 colors support of ConEmu.
```

Comment:
Rewording suggestion:

> Controls whether console virtual terminal sequences are processed by libuv or console.
>
> Useful in particular for enabling ConEmu support of ANSI X3.64 and Xterm 256 colors. Otherwise Windows10 consoles are usually detected automatically.